src/Symfony/Component/Mailer/Transport/Smtp/EsmtpTransport.php

```diff
@@ -183,7 +183,7 @@ private function handleAuth(array $modes): void
 
         $message = sprintf('Failed to authenticate on SMTP server with username "%s" using the following authenticators: "%s".', $this->username, implode('", "', $authNames));
         foreach ($errors as $name => $error) {
-            $message .= sprintf(' Authenticator "%s" returned "%s".', $name, $error);
+            $message .= sprintf(' Authenticator "%s" returned "%s".', $name, $error->getMessage());
```

Comment:
better do this on L176, isn't it?

Reply:
Hi Nicolas, yeah, i agree - it's better to do it there because we can be sure to have an exception within a try-catch block and the comment suggests that only the message is kept, too.
I have moved it there.